crates/sui-core/src/execution_driver.rs

```diff
@@ -71,11 +71,6 @@ pub async fn execution_process(
         let digest = *certificate.digest();
         debug!(?digest, "Pending certificate execution activated.");
 
-        // Process any tx that failed to commit.
-        if let Err(err) = authority.process_tx_recovery_log(None, &epoch_store).await {
-            error!("Error processing tx recovery log: {:?}", err);
-        }
```

Comment:
In the past, this has been necessary, due to client-side disconnects (which can cause txns to be left partially executed without a crash having happened). That specific bug has been fixed, but are we sure there are no more such cases?

Reply:
I believe this should not happen anymore, because
1. `handle_certificate()` kicks off a separate task to make sure execution can continue when client disconnects / cancels: https://github.com/MystenLabs/sui/blob/4a3cfccdd000dcb40c07b299598204598e711da2/crates/sui-core/src/authority_server.rs#L436-L444
2. User submitted certs have to enqueue in TransactionManager, then get executed asynchronously.

Reply:
Yeah I know the known cases are fixed. In any case I think i'm ok with this for now, we can always revert if necessary.